include/vrt.h

```diff
@@ -824,6 +824,10 @@ void VRT_VSM_Cluster_Destroy(VRT_CTX, struct vsmw_cluster **);
 #ifdef va_start	// XXX: hackish
```

Comment:
side note: I'm not sure this `ifdef` is justified anymore? There's no alternative for `VRT_VSC_Alloc` if it isn't defined, and we systematically rely on that function, don't we?